docs/includes/ksql-includes.rst

```diff
@@ -67,7 +67,7 @@ Create and produce data to the Kafka topics ``pageviews`` and ``users``. These s
 -------------------
 Launch the KSQL CLI
 -------------------
-To launch the CLI, run the following command. It will route the CLI logs to the ``./ksql_logs`` directory. By default,
+To launch the CLI, run the following command. It will route the CLI logs to the ``.ksql_logs`` directory. By default,
```

Comment:
@apurvam : This change should not have gone in.  I suppose that was caused by re-branching this PR.

Reply:
@joel-hamill had a PR earlier today/yesterday to fix this text. https://github.com/confluentinc/ksql/pull/1156/files